build.sh

```diff
@@ -28,7 +28,14 @@ find_prunes() {
 }
 
 find_cmd() {
-  echo "find . -type f -and \( -perm +x -or -name '*.sh' \) $(find_prunes)"
+  # GNU `find` dropped compatibility support for the `+` syntax with 4.5.12 (release 2013), it
+  # instead uses `/` which has been supported for a long time. BSD `find` however still used `+`.
+  local perm_format_specifier="/"
+  if [ "$(uname -s)" = "Darwin" ]; then
```

Comment:
@caarlos0 Good point!
I’ll update the PR. It’ll be a little bit more involved since BSD find doesn’t accept `--version` option :disappointed: